src/components/top-features.js

```diff
@@ -0,0 +1,51 @@
+          </g>
+        </svg>
+        <p class="cmp-percentage-card__label font-label">Color System</p>
+        <mark class="cmp-percentage-card__percentage">92%</mark>
+      </div>
+      <div class="cmp-percentage-card">
+        <svg class="cmp-percentage-card__icon" width="194" height="250" viewBox="0 0 194 250">
+          <path fill="#15CBF4" fill-opacity="0.1" fill-rule="evenodd" stroke="#15CBF4" stroke-width="2" d="M166.2 209.1L166 96.6l-32.8 1.6h-65v-5.5c0-21 1.5-42 4.5-50.2 2-5.6 5.8-10.7 11.3-15.1a25.3 25.3 0 0 1 15.7-6.5c4.3 0 8 .7 10.7 2.2 3.8 2.2 7.6 5.9 11.2 11a82.3 82.3 0 0 0 15.5 18.6c3.6 2.6 7.9 4 12.8 4 4.8 0 9-1.8 12.5-5.4a18 18 0 0 0 5.2-13c0-5.5-2.3-11.4-6.9-17.4a46.8 46.8 0 0 0-20.3-14.5A78.2 78.2 0 0 0 111 1a81.5 81.5 0 0 0-42.4 11.2 72.6 72.6 0 0 0-28.4 30.7c-6 12-5.5 34.7-5.9 55.3H2v18.4h32.3v83.6c0 18.5-5.6 23.5-7.8 26.1-3 3.7-8.4 7.5-15.5 7.5H1v14.5h99.3v-14.5h-1.8c-23.5 0-30.1-2.5-30.1-31.1v-86.1h36c19.3 0 23.7.9 24.7 1.3 1.2.4 2.7 1.2 3.3 2.3.3.5 1.4 6.5 1.4 21.4v64.6c0 15-1.9 20-3 21.7-2.4 3.7-5.2 5.7-14 6v14.4h75.6V234c-23.6 0-26.2-2.5-26.2-24.8z"/>
+        </svg>
+        <p class="cmp-percentage-card__label font-label">Typography System</p>
+        <mark class="cmp-percentage-card__percentage">89%</mark>
+      </div>
+      <div class="cmp-percentage-card">
+        <svg class="cmp-percentage-card__icon" width="222" height="267" viewBox="0 0 222 267">
+          <path fill="#F3C448" fill-opacity="0.1" stroke="#F3C448" stroke-width="2" d="M217 1H5a4 4 0 0 0-4 4v257a4 4 0 0 0 4 4h212a4 4 0 0 0 4-4V5a4 4 0 0 0-4-4"/>
+          <path fill="#F3C448" fill-opacity="0.1" stroke="#F3C448" stroke-width="2" d="M191 26a10 10 0 0 1 10 10v25a10 10 0 0 1-10 10H31a10 10 0 0 1-10-10V36a10 10 0 0 1 10-10h160zM71 91a10 10 0 0 1 10 10v40a10 10 0 0 1-10 10H31a10 10 0 0 1-10-10v-40a10 10 0 0 1 10-10h40zm0 80a10 10 0 0 1 10 10v40a10 10 0 0 1-10 10H31a10 10 0 0 1-10-10v-40a10 10 0 0 1 10-10h40z"/>
+          <path fill="#F3C448" d="M67.5 102.2a4 4 0 0 1 5.6 0l1.4 1.4a4 4 0 0 1 0 5.6L49.8 134l-1.4 1.4a4 4 0 0 1-5.7 0l-1.4-1.4-14.1-14.2a4 4 0 0 1 0-5.6l1.4-1.4a4 4 0 0 1 5.6 0L45.6 124l21.9-22zm0 82.4a4 4 0 0 1 5.6 0l1.4 1.5a4 4 0 0 1 0 5.6l-24.7 24.8-1.4 1.4a4 4 0 0 1-5.7 0l-1.4-1.4-14.1-14.2a4 4 0 0 1 0-5.6l1.4-1.5a4 4 0 0 1 5.6 0l11.4 11.4 21.9-22zm126-68.6h-85a7.5 7.5 0 0 1 0-15h85a7.5 7.5 0 0 1 0 15zm-20 25h-65a7.5 7.5 0 0 1 0-15h65a7.5 7.5 0 0 1 0 15zm20 55h-85a7.5 7.5 0 0 1 0-15h85a7.5 7.5 0 0 1 0 15zm-20 25h-65a7.5 7.5 0 0 1 0-15h65a7.5 7.5 0 0 1 0 15z"/>
+        </svg>
+        <p class="cmp-percentage-card__label font-label">Form Components</p>
+        <mark class="cmp-percentage-card__percentage">88%</mark>
+      </div>
+    </div>
+    <ul class="cmp-percentage-list">
+      <li class="cmp-percentage-list__item">Navigation components <mark class="cmp-percentage-list__percentage">79%</mark></li>
```

Comment:
I feel like you can get away with your `util-bold` class instead of `cmp-percentage-list__percentage`